server/service/src/invoice/outbound_shipment/insert/generate.rs

```diff
@@ -32,6 +32,7 @@ pub fn generate(
         status: InvoiceRowStatus::New,
         on_hold: input.on_hold.unwrap_or(false),
         colour: input.colour,
+        mode: InvoiceRowMode::Store,
```

Comment:
OG saves shipments/requisitions in store mode even if the store is a dispensary.